symbols.xml

```diff
@@ -127,90 +127,90 @@
 
         .data
 
-        <global-address name='cursor'/>
-        <global-address name='selection_rect'/>
-        <global-address name='gamemode'/>
-        <global-address name='gametype'/>
+        <global-address name='cursor'value='0x00f93040'/>
```

Comment:
No spaces before `value`? I'm assuming DFHack doesn't choke on this, but it would be nice for readability purposes to have spaces.